app/lib/account/default_auth_provider.dart

```diff
@@ -112,7 +112,7 @@ class DefaultAuthProvider extends BaseAuthProvider {
           oauth2_v2.Oauth2Api.userinfoEmailScope,
           oauth2_v2.Oauth2Api.userinfoProfileScope,
         ].join(' '),
-        'state': state,
+        'state': Uri(queryParameters: state).toString().substring(1),
```

Comment:
Should we not do something a bit more sane.

Like: `json + base64` or something?

Reply:
Done.